### PR TITLE
cppcheck performance fixes in linux/

### PR DESCRIPTION
**Pull request by tobbi**

### CODE REVIEW

File: xbmc/linux/OMXClock.cpp [hunk 1]
```diff
@@ -209,14 +208,13 @@ bool  OMXClock::OMXStop(bool lock /* = true */)
...
 
-  OMX_ERRORTYPE omx_err = OMX_ErrorNone;
   OMX_TIME_CONFIG_CLOCKSTATETYPE clock;
   OMX_INIT_STRUCTURE(clock);
 
   clock.eState      = OMX_TIME_ClockStateStopped;
   clock.nOffset     = ToOMXTime(-1000LL * OMX_PRE_ROLL);
 
-  omx_err = m_omx_clock.SetConfig(OMX_IndexConfigTimeClockState, &clock);
+  MX_ERRORTYPE omx_err = m_omx_clock.SetConfig(OMX_IndexConfigTimeClockState, &clock);
```
Comment (anaconda) [inline xbmc/linux/OMXClock.cpp]: Typo.

**Comment by tobbi** [comment]
Typo should be fixed.

File: xbmc/linux/LinuxTimezone.cpp [hunk 2]
```diff
@@ -197,13 +197,13 @@ std::string CLinuxTimezone::GetCountryByTimezone(const std::string timezone)
...
-  
 #if !defined(TARGET_DARWIN)
-  use_timezone = true;
+  bool use_timezone = true;
 #else
   if (g_sysinfo.IsAppleTV2())
-    use_timezone = true;  
+    bool use_timezone = true;
+  else
+    bool use_timezone = false;
```
Comment (notspiff) [inline xbmc/linux/LinuxTimezone.cpp]: it is incorrect, variable goes out of scope immediately..

Comment (tobbi) [inline xbmc/linux/LinuxTimezone.cpp]: Actually, I will remove that part. It was about not using the previous value of that variable.

**Comment by tobbi** [comment]
Fixed the timezone if statement. Please check again!

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by MartijnKaijser** [comment]
Any other comments before merging?

**Comment by Jalle19** [comment]
@MartijnKaijser don't think so

**Pull request merged**